src/Controllers/expenses.controller.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+const service = require('../services/expenses.services');
+const userService = require('../services/users.services');
+
+const getExpenses = (req, res) => {
+  const {
+    userId,
+    from,
+    to,
+  } = req.query;
+
+  const normalizedUrl = new URL('http://localhost:3000/' + req.url);
+  const categories = normalizedUrl.searchParams.getAll('categories');
+
+  let expenses = service.getAll();
+
+  if (userId) {
+    expenses = expenses.filter(
+      expense => expense.userId === +userId
```

Comment:
```suggestion
      expense => expense.userId === Number(userId)
```

Reply:
The same in all other places, it is easier for reading 🙂

Reply:
i had it this way before, but switched to + to make is more simple)))
i guess i should bring everything back))